go.mod

```diff
@@ -15,6 +16,7 @@ require (
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/timandy/routine v1.1.5 // indirect
```

Comment:
I think you need `go mod tidy`, since this is not an indirect but actually imported.

Also, perhaps look into [github.com/petermattis/goid](https://github.com/petermattis/goid) instead: it's a much smaller dependency. (It's what we used at Square; Peter Mattis actually worked at Square briefly before starting CockroachDB.)

Also, this has changed in slight and subtle ways in past versions of Go. I know this shouldn't get used in code bases that _import_ protocompile, since it's behind an unexported flag. But perhaps it would still be safer to include it via a build tag: so most builds will never pull this in as a dependency at all, but we can run tests with a special build tag that pulls it in.